samples/apps/pipes.jakt

```diff
@@ -0,0 +1,183 @@
+
+enum Stream : c_int {
+    STDIN  = 0,
+    STDOUT = 1,
+    STDERR = 2
+}
+
+struct Pipe {
+    reading_end: c_int
+    writing_end: c_int
+}
+
+class System {
+    public function getErrno() -> c_int {
+        unsafe { cpp {
+            "return errno;"
+        }}
+        // Unreachable code
+        return -1
+    }
+
+    public function pipe() throws -> Pipe  {
+        unsafe{
```

Comment:
Not nearly all of this needs to be wrapped in unsafe. In fact, it looks like *only* the cpp block needs it.

Reply:
Also missing a space between the keyword and {